ember.md

```diff
@@ -1,27 +1,86 @@
 # Ember
-1. [General Structure](#gstructure)
+1. [General Structure](#general-structure)
+1. [Destructuring](#destructuring)
 1. [CSS](#css)
 
 ## General Structure
-<a name="gstructure"></a><a name="1.1"></a>
 
-- [1.1](#gstructure) **Property Order**: For components and controllers, follow this order for properties and methods
+- 1.1 **Property Order**: For components and controllers, follow this order for properties and methods
   + **Properties**
     + Put ember specific properties (e.g., `classNames`, `tagNames`) before custom properties (e.g., `someSpecificAshProp : true`)
   + Component lifecycle hooks (in order) <a href="https://guides.emberjs.com/v2.7.0/components/the-component-lifecycle/">see order in documentation</a>
   + Custom methods
   + `actions` go last
 
 
+## Destructuring
+Extract multiple values from data stored in objects and arrays.
+> Why? Prior to ES2015, there was not way to extract muliple property values from one object at once. Enter `Destructuring`.
```

Comment:
This `why` doesn't really explain why destructuring is useful within our Ember apps. Should talk about importing only what you need, and not importing the entire `Ember` object every time you need a small piece of the Ember object